make/common.Makefile

```diff
@@ -1,3 +1,4 @@
+PYTHON_EXECUTABLE ?= python2.7
```

Comment:
Should be `PYTHON2_EXECUTABLE` (here and elsewhere) so no one will use it by mistake for new python code